driver/src/main/com/mongodb/client/MongoCollection.java

```diff
@@ -45,6 +45,12 @@
  *
  * <p>Note: Additions to this interface will not be considered to break binary compatibility.</p>
  *
+ * <p>MongoCollection is generic allowing for different types to represent {@code Documents}. Any custom classes must have a
```

Comment:
"Documents" shouldn't be in an @code block since it's not code (there's no Documents class).